lms/templates/instructor/instructor_dashboard_2/certificate-bulk-white-list.underscore

```diff
@@ -5,10 +5,11 @@
     </p>
     <form id="bulk-white-list-exception-form" enctype="multipart/form-data">
       <div class="customBrowseBtn">
-        <input disabled="disabled" class="browse-file" placeholder="<%- gettext("Choose File") %>" />
+        <label for="bulk-exception-upload" class="sr"><%- gettext('Choose CSV file') %></label>
```

Comment:
In this case, you removed the "disabled" property. In certificate-invalidation.underscore, you did not.